source/common/runtime/runtime_features.cc

```diff
@@ -93,6 +93,8 @@ FALSE_RUNTIME_GUARD(envoy_reloadable_features_runtime_initialized);
 // TODO(mattklein123): Also unit test this if this sticks and this becomes the default for Apple &
 // Android.
 FALSE_RUNTIME_GUARD(envoy_reloadable_features_always_use_v6);
+// TODO(nezdolik): Flip this to true to multiplex eds by default.
+FALSE_RUNTIME_GUARD(envoy_reloadable_features_multiplex_eds);
```

Comment:
Is there any reason not to have this be true by default?

Reply:
it makes sense, am just being overcautious

Reply:
If you can test it in prod ahead of time, I would flip to true by default. If not, I think it's fine to leave as false and come back and switch to true after you test it.